bin/tokenizer

```diff
@@ -27,6 +28,6 @@ while tokenizer.has_more_tokens?
     token_type = "symbol"
   end
 
-  puts("  <#{token_type}> #{token} </#{token_type}>")
+  puts("  <#{token_type}> #{CGI.escapeHTML(token)} </#{token_type}>")
```

Comment:
As discussed, we’re emitting XML rather than HTML, but XML and HTML have similar escaping rules so `CGI.escapeHTML` should work for our purposes.